lib/spack/spack/test/packages.py

```diff
@@ -77,30 +75,20 @@ def test_package_class_names(self):
     #
 
     def test_import_package(self):
-        import spack.pkg.builtin.mock.mpich
+        pass
```

Comment:
@tgamblin Good catch! I guess the IDE import optimizer didn't like the import as the only statement in a function. I'll fix this and checkout the rest of the tests.